doc/news/changes/minor/20170314DenisDavydov

```diff
@@ -0,0 +1,4 @@
+New: extend internal::VectorOperations::parallel_for() and
+internal::VectorOperations::parallel_reduce() to [start,end).
+<br>
+(Denis Davydov, 2017/03/14)
```

Comment:
Since this is an internal interface, my preference would be not to explicitly mention it here, unless it has effect on user-visible code such as performance improvements - but here I guess it rather helps the development down the road...